TouchPortal-API/TouchPortalAPI/__init__.py

```diff
@@ -20,40 +21,84 @@ class TYPES:
 class Client(EventEmitter):
     TPHOST = '127.0.0.1'
     TPPORT = 12136
+    RCV_BUFFER_SZ = 4096   # [B] incoming data buffer size
+    SND_BUFFER_SZ = 32**4  # [B] maximum size of send data buffer (1MB)
+    SLEEP_PERIOD = 0.01    # [s] event loop sleep between socket read events
+    SOCK_EVENT_TO = 1.0    # [s] timeout for selector.select() event monitor
```

Comment:
Note that since writing to a socket is almost always possible, this TO doesn't typically sleep the process. This is really a "worst case" timeout in case the socket is blocked from writing for some reason. So the `SLEEP_PERIOD` is still the main control over how long the main loop sleeps/yields to others threads.